Lib/importlib/_bootstrap_external.py

```diff
@@ -140,13 +160,19 @@ def _path_isdir(path):
     return _path_is_mode_type(path, 0o040000)
 
 
-def _path_isabs(path):
-    """Replacement for os.path.isabs.
+if _MS_WINDOWS:
+    def _path_isabs(path):
+        """Replacement for os.path.isabs."""
+        if not path:
+            return False
+        root, _ = _path_splitroot(path)
+        return ((root.startswith("\\\\") or root.endswith(tuple(path_separators)))
```

Comment:
If the split root value is expanded to support mixed slashes, which it can be, then this could be changed to replace slashes with backslashes in the split root before checking the value:

```python
root = _path_splitroot(path)[0].replace('/', '\\')
return root.startswith('\\\\') or root.endswith('\\')
```